src/VisualStudio/Core/Def/Venus/ContainedDocument.cs

```diff
@@ -90,6 +90,7 @@ public static ContainedDocument TryGetContainedDocument(DocumentId id)
         private readonly VisualStudioProject _project;
 
         public bool SupportsRename { get { return _hostType == HostType.Razor; } }
+        public bool SupportsSemanticSnippets { get { return _hostType == HostType.Razor; } }
```

Comment:
We want this to be `!=`, right? Or perhaps `return false` is the safer bet?

Reply:
that is correct, my bad